pkg/deepcopy/testdata/cronjob_types.go

```diff
@@ -153,8 +153,13 @@ type SpecificCases struct {
 
 	// Case: kubernetes-sigs/controller-tools#262 part 2 (see type definition)
 	StringMap MapOfStrings `json:"stringMap"`
+
```

Comment:
I'm a bit confused that this addition doesn't lead to any necessary change in testdata

Reply:
IIUC, it is covered by this existing assignment because it is a string field.

https://github.com/kubernetes-sigs/controller-tools/blob/12ea13fd95e06878edb3b2febb5674e05bc73151/pkg/deepcopy/testdata/zz_generated.deepcopy.go#L572-L573

Reply:
Makes sense